### PR TITLE
make tests pass on 46970-milestone-2

### PR DESCRIPTION
**Pull request by npretto**
We'll need to revert https://github.com/metabase/metabase/pull/48293/commits/0888589d373d694a715582da2c30f8bed2abb8cb before we merge the integration branch on master, but for now this should make make CI green, allowing us to focus on real failures

This should make ci green on https://github.com/metabase/metabase/pull/48069

### CODE REVIEW

File: e2e/support/helpers/api/updateSetting.ts [hunk 1]
```diff
@@ -7,5 +7,10 @@ export const updateSetting = <
   setting: TKey,
   value: TValue,
 ): Cypress.Chainable<Cypress.Response<never>> => {
+  // TODO: remove before merging integration branch
+  if (setting === "enable-embedding-static") {
+    updateSetting("enable-embedding", value as boolean);
```
Comment (heypoom) [inline e2e/support/helpers/api/updateSetting.ts:12]: Is this to make embedding work with the current backend?

**Review by heypoom — APPROVED** [review]

File: e2e/support/helpers/api/updateSetting.ts (hunk 1 above)
Comment (npretto) [reply e2e/support/helpers/api/updateSetting.ts:12]: exactly, without these small changes we'd have all tests fail and we won't be able to see the real failures among the ones caused by the FE already being on the new settings while the BE is not

Comment (heypoom) [reply e2e/support/helpers/api/updateSetting.ts:12]: I see, it all makes sense now. Thanks!

**Review by oisincoveney — APPROVED** [review]
lgtm, let's get this in

**Pull request merged**